Dockerfile

```diff
@@ -10,4 +12,5 @@ EXPOSE 4443
 
 WORKDIR /go/src/github.com/docker/notary
 
-CMD [ "notary-server", "-config", "cmd/notary-server/dev-config.json" ]
+
+CMD [ "notary-server", "-config", "cmd/notary-server/stage-config.json" ]
```

Comment:
Perhaps

```Dockerfile
CMD [ "--config", "cmd/notary-server/stage-config.json" ]
ENTRYPOINT [ "notary-server" ]
```

That way the config can be overridden from the command-line, e.g.

    docker run notary-server -config foobar.json